pkg/quarkus/v1alpha/scaffolds/init.go

```diff
@@ -79,9 +79,9 @@ func (s *initScaffolder) Scaffold() error {
 		&templates.ApplicationPropertiesFile{
 			ProjectName: s.config.GetProjectName(),
 		},
-		&templates.Makefile{
-			Image:            "",
-			KustomizeVersion: "v3.5.4",
-		},
+		// &templates.Makefile{
+		// 	Image:            "",
+		// 	KustomizeVersion: "v3.5.4",
+		// },
```

Comment:
Why did we remove it from here? this means that when I run `operator-sdk init --plugins quarkus --domain example.com --project-name memcached-quarkus-operator` I do NOT get a `Makefile`. I *MUST* run `operator-sdk create api` to see one. I don't think that makes sense. Once I initialize a project I should automatically have the `Makefile`. 

What was the reason to move it?

Reply:
The reason behind this change is we need a version and group name for the identification of files located in the `/target/kubernetes` directory. That info we will get it when we go for `create api`

Reply:
Added back the Makefile in its old place.